graphql_compiler/tests/test_compiler.py

```diff
@@ -4237,12 +4318,71 @@ def test_fold_same_edge_type_in_different_locations(self):
                   schema_1."Animal" AS "Animal_5"
                 JOIN schema_1."Animal" AS "Animal_6" ON "Animal_5".uuid = "Animal_6".parent
                 GROUP BY
-                  "Animal_5".uuid
+                    "Animal_5".uuid
```

Comment:
Why the whitespace-only change? This is not what the other `GROUP BY` clauses look like. Overall, the entire query uses a mix of 2-space and 4-space indentation, and changing just this one location doesn't make sense. I'd either fix the whole thing or not change anything in it.